recipes/faac/all/conanfile.py

```diff
@@ -70,54 +78,114 @@ def requirements(self):
 
     def validate(self):
         if is_msvc(self):
-            # FIXME: add msvc support since there are MSBuild files upstream
-            raise ConanInvalidConfiguration("libfaac conan-center recipe doesn't support building with Visual Studio yet")
+            if self.settings.arch not in ["x86", "x86_64"]:
+                raise ConanInvalidConfiguration(f"{self.ref} only supports x86 and x86_64 with Visual Studio")
+            if self.options.drm and not self.options.shared:
+                raise ConanInvalidConfiguration(f"{self.ref} drm can't be built as static with Visual Studio")
```

Comment:
drm?

Reply:
The option couldn't be built as static? Not sure this new sentence makes lot of sense ;)

Reply:
Attempt two! lol

```suggestion
                raise ConanInvalidConfiguration(f"{self.ref}'s option `drm` can't be built as static with Visual Studio")
```

Although you'd need to pass both to clearly help with the options that complicated since we dont know which the sure set versus the defaults 🤔 too hard for my brain

Reply:
please ignore my bad typing, still on coffee cup too 🍧